icepyx/core/read.py

```diff
@@ -742,8 +754,33 @@ def load(self):
         # so to get a combined dataset, we need to keep track of spots under the hood, open each group, and then combine them into one xarray where the spots are IDed somehow (or only the strong ones are returned)
         # this means we need to get/track from each dataset we open some of the metadata, which we include as mandatory variables when constructing the wanted list
 
+        if not self.vars.wanted:
+            raise AttributeError(
+                'No variables listed in self.vars.wanted. Please use the Variables class '
+                'via self.vars to search for desired variables to read and self.vars.append(...) '
+                'to add variables to the wanted variables list.'
+            )
+        
+        # Append the minimum variables needed for icepyx to merge the datasets
+        var_list=[
+            "sc_orient",
+            "atlas_sdp_gps_epoch",
+            "cycle_number",
+            "rgt",
+            "data_start_utc",
+            "data_end_utc",
+        ]
+        
+        # Adjust the nec_varlist for individual products
+        if self.product == "ATL11":
+            var_list.remove("sc_orient")
+        
+        # Note: This fails if we are reading a file that doesn't have the var_list
+        # variables in it.
```

Comment:
Good catch. In looking through the code, it seems there are built-in fixes for any of these that are missing (and the user is warned they are happening). We could put it in a try-except if we make sure that `load()` really will work without any of them.

Reply:
This should now be resolved with the required variables check by product.